src/Contract/Exclusion/ExclusionCheckInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Contract\Exclusion;
+
+use PhpParser\Node;
+use Rector\Contract\Rector\PhpRectorInterface;
+
+interface ExclusionCheckInterface
```

Comment:
Why interface?

Reply:
To be implemented by anyone wanting to write their own exclusion check, for example the one checking also checking for other skip docblocks from other tools

Reply:
That's what `ExclusionCheckInterface` does

Reply:
Ah, my bad. I thought it's interface for `ExclusionManager`

Allright :+1: